src/components/Header/index.tsx

```diff
@@ -15,9 +16,15 @@ interface HeaderProps {
 }
 
 const Header: FC<HeaderProps> = ({ pageName, hasBackIcon = false }) => {
+  const previousPage = -1;
+  const navigate = useNavigate();
```

Comment:
local constant out of function or at least swap places with hook

Reply:
fixed